wunderkraut/WunderTools
# Feature/codeception

Adds Codeception support:

1. Visual, Functional, Unit, Acceptance tests.
2. Per environment settings for tests.
3. Default PhantomJS browser load on test startup.
4. new command: ./build.sh test.

To test:
1. Build new drupal instalation
2. Make sure url's are correct in tests/suitname.suite.yml
3. From drupal directory run ./build.sh test
